modules/parent-join/src/yamlRestTest/resources/rest-api-spec/test/30_inner_hits.yml

```diff
@@ -140,7 +140,7 @@ profile fetch:
   - gt: { profile.shards.0.fetch.breakdown.next_reader: 0 }
   - gt: { profile.shards.0.fetch.breakdown.load_stored_fields_count: 0 }
   - gt: { profile.shards.0.fetch.breakdown.load_stored_fields: 0 }
-  - match: { profile.shards.0.fetch.debug.stored_fields: [_id, _ignored, _routing, _source] }
+  - match: { profile.shards.0.fetch.debug.stored_fields: [_id, _routing, _source] }
```

Comment:
I wonder if with this change, the skip above needs updating? Isn't it surprising that this test runs? 8.14 returns the `_ignored` field while 8.15 does not?

Reply:
I fixed this but the other was ok.